Gemfile

```diff
@@ -1,3 +1,5 @@
 source 'https://rubygems.org'
 
-gem 'graphql_swift_gen', path: 'Dependencies/Swift Gen'
\ No newline at end of file
+git 'https://github.com/Shopify/graphql_swift_gen', branch: 'task/buy-sdk-3' do
```

Comment:
Neat. What is the advantage of doing this over the git sub module? 
nit: who maintains the `task/buy-sdk-3` branch on the gem's repo?

Reply:
Dima maintains that branch, it's what the current submodule points at. Some versions of bundler don't like relative paths, this was easier than figuring out how to change the github bundler version.

In general I think we should move away from having a git submodule as a dependency, but for now our build and compile also depends on a file in that repo so I can't remove it yet :(